examples/hypervisor_cluster_profile.go

```diff
@@ -0,0 +1,112 @@
+package main
+
+import (
+	"fmt"
+	"github.com/HewlettPackard/oneview-golang/ov"
+	"github.com/HewlettPackard/oneview-golang/utils"
+	"os"
+)
+
+func main() {
+	var (
+		clientOV                *ov.OVClient
+		hcp_name                = "test"
+		new_hcp                 = "test_new"
+		server_profile_template = utils.Nstring("/rest/server-profile-templates/278cadfb-2e86-4a05-8932-972553518259")
+		hypervisor_manager      = utils.Nstring("/rest/hypervisor-managers/1ded903a-ac66-41cf-ba57-1b9ded9359b6")
+		/*		sp_sn       = "VCGRE1S007"
```

Comment:
Please add a note on why you commented this. Otherwise, remove the commented section.

Reply:
Done